tests/foreman/cli/test_errata.py

```diff
@@ -495,6 +495,52 @@ def test_positive_list_affected_chosts(self):
                 virtual_machine.hostname in result
             ), "VM host name not found in list of applicable hosts"
 
+    @tier3
+    def test_install_errata_to_one_host(self):
+        """Install an erratum to one of the hosts in a host collection.
+
+        :id: bfcee2de-3448-497e-a696-fcd30cea9d33
+
+        :expectedresults: Errata was successfully installed in only one of the hosts in
+        the host collection
+
+
+        :Setup: Errata synced on satellite server.
+
+        :Steps:
+           1. Remove FAKE_2_CUSTOM_PACKAGE_NAME packages from one host
+           2. host-collection erratum install --errata <errata> --id <id>
+            --organization <org name>
+           3. Assert first host does not have any FAKE_2_CUSTOM_PACKAGE_NAME packages.
+           4. Assert second host does have FAKE_2_CUSTOM_PACKAGE
+
+        :expectedresults: Erratum is only installed on one host.
+
+        :BZ: 1810774
+        """
+        # Remove package on first VM to remove need for CUSTOM_ERRATA_ID
+        result = ssh.command(
+            f'yum erase -y {self.CUSTOM_PACKAGE}', self.virtual_machines[0].ip_addr
+        )
```

Comment:
`VirtualMachine` objects provide `run` method exactly so you don't have to extract ip address of machine manually.

```suggestion
        result = self.virtual_machines[0].run(f'yum erase -y {self.CUSTOM_PACKAGE}')
```

Reply:
OK, change made, thank you